src/Slicer.cpp

```diff
@@ -0,0 +1,87 @@
+//
+// Created by martin on 2017-05-03.
+//
+
+#include "../include/Slicer.h"
+
+/**
+ * Convenience function for checking if slicing is enabled (if boost version >= 1.64 was
+ * found by cmake).
+ * @return true if slicing is possible, false otherwise
+ */
+bool Slicer::enabled() {
```

Comment:
Sådana här funktioner som returnerar booleans brukar väl börja med "is" så den borde heta "is_enabled" tycker jag.